apps/condo/domains/user/schema/SendMessageToSupportService.test.js

```diff
@@ -122,7 +123,44 @@ describe('SendMessageToSupportService', () => {
 
         const messages = await Message.getAll(adminClient, { id: result.id })
         expect(messages).toHaveLength(1)
-        expect(messages[0].meta.organizationsData).toHaveLength(2)
+        expect(messages[0].meta.residentsExtraInfo).toHaveLength(2)
+    })
+
```

Comment:
Please add a test where the service consumer's organization differs from the resident organization.

Reply:
done